_data/hosting_providers.json

```diff
@@ -2,12 +2,12 @@
   {
     "name": "Snel.com",
     "link": "https://www.snel.com",
-    "category": "partial",
+    "category": "full",
     "tutorial": "https://www.snel.com/support/set-up-ssl-for-my-website/",
-    "announcement": "",
+    "announcement": "https://www.snel.com/webhosting/",
```

Comment:
This is a bit of a nitpick, but I think this should probably be the link for `plan` and not `announcement`. The only difference is the text used in the "relevant link" column at https://certbot.eff.org/hosting_providers, but I think this link describes the plans that offer automated TLS and not a special announcement of automated TLS support.